db/migrate/20170608155514_add_training_session_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTrainingSessionIdToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :training_session_id, :string
```

Comment:
I'm not sure this is going to work the way you want. This means that each user can only ever be in a single training session. I'm also not sure that having training information exist on the user is good design because it breaks encapsulation. The training session model depends on users, but the user doesn't need to have training sessions.

If you want to track the users in a training session you will probably need to create a [Join table](https://en.wikipedia.org/wiki/Associative_entity) that only has 2 columns: user_id and training_session_id OR you can use [has_many :through](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association) and use certification as the 'join table'